### DISCUSSION TITLE
Exclude one item from collection

### DISCUSSION THREAD

**Discussion by qbixx** [statamic/cms#5692] [Q&A]
I'm currently working on a news section. A news item can be 'highlighted' or not. When an article is highlighted, the newest one will be shown at the top of the page and will be bigger than the regular items. Older highlighted item, it will show just like the regular items in a blog overview.

I'm using the following code to get the latest highlighted item:
```{{ collection:nieuws highlighted:is="true" limit="1" sort="date:asc" as="news" }}```

I'm building the regular blog posts with the code below:
```{{ collection:nieuws paginate="20" as="news" }}```

When I use the code above the latest highlighted item will be shown twice on the page (once as highlighted item and once in the regular blog overview). How can I make sure that the latest highlighted item doesn't show up in the regular blog posts overview?

**Comment by duncanmcclean** [comment]
When you're looping through, you could just check if it's highlighted?

```antlers
{{ collection:nieuws paginate="20" as="news" }}
  {{ if highlighted === false }}
    <!-- News card -->
  {{ /if }}
{{ /collection:nieuws }}
```

**Comment by qbixx** [comment]
Thanks, this would exclude all highlighted items. Ideally only the latest highlighted item isn't included. The older highlighted items should be added to the blog overview.

**Reply by duncanmcclean** [reply]
Oh okay. If you're using Statamic 3.3, maybe you could store the ID of the highlighted news entry as a variable in Antlers, then check when looping through that the ID isn't the one in the variable.

Variable docs: https://statamic.dev/new-antlers-parser#variables

> Note: you'll need to use the new `runtime` Antlers parser for this to work.

**Reply by qbixx** [reply]
Thanks, I will check this out. The `{{ if highlighted === false }}` otherwise would be a nice alternative 👍

**Reply by Alpenverein** [reply]
Did it work? I am trying to do something simular. But I have problems to read/write variables between the scopes of diffenrent loops..

**Reply by qbixx** [reply]
I used the `{{ if highlighted === false }}` option.

**Comment by austriker27** [comment]
Random idea, I think if you are using the new antlers parser you could store the id of the latest, highlighted entry at the top of the file and then exclude that one from your second loop I think. 

```handlebars 

{{ latest_highlighted = '' }}

{{ collection:nieuws highlighted:is="true" limit="1" sort="date:asc" as="news" }}
  {{ latest_highlighted = id }}
{{ /collection:nieuws }}

{{ collection:nieuws paginate="20" as="news" :id:not="latest_highlighted" }}
{{ /collection:nieuws }} 
```

**Comment by jackmcdade** [comment]
```
{{ unless first && highlighted }}
// everything
{{ /unless }}
```
